### ISSUE TITLE
`no-indented-block` (`E112`) false positive on docstring followed by comment

### ISSUE THREAD

**Issue by DetachHead** [astral-sh/ruff#6930]
```py
def asdf():
    """asdf"""  # asdf

```
```
main.py:3:1: E112 Expected an indented block
```

**Comment by charliermarsh** [comment]
This was fixed at some point -- no longer triggers on `main`.
